pkg/proxy/redirect.go

```diff
@@ -15,9 +15,10 @@
 package proxy
 
 import (
-	"github.com/cilium/cilium/pkg/revert"
 	"time"
 
+	"github.com/cilium/cilium/pkg/revert"
```

Comment:
TODO: fix import order.

Reply:
^^^

(If you set up your editor to run `go fmt` on save, then you can just put it in the same block as the rest of the cilium bits and it will resolve this for you..)